third_party/renderman-24/plugin/hdPrman/CMakeLists.txt

```diff
@@ -22,8 +22,14 @@ endif()
 set(optionalIncludeDirs "")
 set(optionalPublicClasses "")
 if (${PXR_ENABLE_MATERIALX_SUPPORT})
-    list(APPEND optionalLibs ${MATERIALX_LIBRARIES} hdMtlx)
-    list(APPEND optionalIncludeDirs ${MATERIALX_INCLUDE_DIRS})
+    list(APPEND optionalLibs
+        MaterialXCore
+        MaterialXFormat
+        MaterialXGenShader
+        MaterialXGenOsl
+        MaterialXRender
```

Comment:
I'm curious as to why there is a dependence on MaterialXRender ?
Generally I would assume this is not required.

Reply:
I grepped for `#include <MaterialX` in the code and added what was found. In this case, I saw [one MaterialXRender include being accessed](https://github.com/PixarAnimationStudios/USD/blob/dev/third_party/renderman-24/plugin/hdPrman/matfiltMaterialX.cpp#L49).